config/crd/bases/appmesh.k8s.aws_gatewayroutes.yaml

```diff
@@ -25,733 +17,743 @@ spec:
-                              type: string
-                            virtualServiceRef:
-                              description: Reference to Kubernetes VirtualService
-                                CR in cluster to associate with the gateway route
-                                virtual service target. Exactly one of 'virtualServiceRef'
-                                or 'virtualServiceARN' must be specified.
-                              properties:
-                                name:
-                                  description: Name is the name of VirtualService
-                                    CR
-                                  type: string
-                                namespace:
-                                  description: Namespace is the namespace of VirtualService
-                                    CR. If unspecified, defaults to the referencing
-                                    object's namespace
-                                  type: string
-                              required:
-                              - name
-                              type: object
-                          type: object
-                      required:
-                      - virtualService
-                      type: object
-                  required:
-                  - target
-                  type: object
-                match:
-                  description: An object that represents the criteria for determining
-                    a request match.
-                  properties:
-                    hostname:
-                      description: The client specified Hostname to match on.
-                      properties:
-                        exact:
-                          description: The value sent by the client must match the
-                            specified value exactly.
-                          maxLength: 253
-                          minLength: 1
-                          type: string
-                        suffix:
-                          description: The value sent by the client must end with
-                            the specified characters.
-                          maxLength: 253
-                          minLength: 1
-                          type: string
-                      type: object
-                    metadata:
-                      description: An object that represents the data to match from
-                        the request.
-                      items:
-                        description: GRPCGatewayRouteMetadata refers to https://docs.aws.amazon.com/app-mesh/latest/APIReference/API_GrpcGatewayRouteMetadata.html
+  versions:
+  - additionalPrinterColumns:
+    - description: The AppMesh GatewayRoute object's Amazon Resource Name
+      jsonPath: .status.gatewayRouteARN
+      name: ARN
+      type: string
+    - jsonPath: .metadata.creationTimestamp
+      name: AGE
+      type: date
+    name: v1beta2
```

Comment:
what is this name?

Reply:
apiextensions v1 lets you have multiple versions of CRDs, so now the versions have names. https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/